WalletWasabi/WabiSabi/Backend/Models/Alice.cs

```diff
@@ -18,6 +19,7 @@ public Alice(Coin coin, OwnershipProof ownershipProof, Round round)
 		}
 
 		public Round Round { get; }
+		public AsyncLock AsyncLock { get; } = new();
```

Comment:
Ah, this is due to the refactoring in #6067, which is not yet in effect here, but basically the goal is to ensure that there is at most one request being processed per alice in the coordinator as a whole, including across rounds.

This is currently moot since the `Arena`'s `AsyncLock` currently linearizes *all* requests everything, but once it returns, this will have a similar effect on the requests associated with a particular coin.

Reply:
So what about:

```csharp
/// <summary>Makes sure that at most one HTTP request is being processed at a time.</summary>
/// <remmarks>The ongoing goal is to get rid of this lock and use ... instead.</remmarks>
```

?

Reply:
there are some comments to that effect in the subsequent refactorings already, so i would rather not introduce more rebasing friction in this specific PR